### PR TITLE
[feature] support read map and struct type

### PR DESCRIPTION
**Pull request by gnehil**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

support read map and struct type

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: spark-doris-connector/src/test/java/org/apache/doris/spark/serialization/TestRowBatch.java [hunk 1]
```diff
@@ -57,7 +51,13 @@
...
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.ImmutableList;
+import java.io.ByteArrayOutputStream;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.hamcrest.core.StringStartsWith.startsWith;
```
Comment (JNSimba) [inline spark-doris-connector/src/test/java/org/apache/doris/spark/serialization/TestRowBatch.java:60]: startwith not use?

Comment (gnehil) [reply spark-doris-connector/src/test/java/org/apache/doris/spark/serialization/TestRowBatch.java:60]: It's used, just adjust the order of import

**Comment by JNSimba** [comment]
ci run failed, please check, thank you

**Comment by gnehil** [comment]
> ci run failed, please check, thank you

done

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**